src/models/Pet.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+
+const PetSchema = new mongoose.Schema(
+  {
+    petName: {
+      type: String,
+      required: [true, 'Please provide pet name'],
+      maxlength: [100, 'Name can not be more than 100 characters'],
+    },
+    petType: {
+      type: String,
+      enum: ['Cat', 'Dog', 'Other'],
+      default: 'Cat',
+      required: [true, 'Please provide type'],
```

Comment:
It is designer decision, of course, but why do you have to set a default here? Is the app primary for adopting cats?